site/sass/components/_card-study.scss

```
@@ -0,0 +1,162 @@
+@use 'sass:math';
+
+.card-study {
+  $break-2-col: 980px;
+  $break-feature-col: 690px;
+  $break-featured-stack: 0 689px;
+  $scale-max: 1600px;
+  $clamp-check: clamp(1px, 1vw, 2px);
+
+  border-radius: 0.625rem;
+  display: grid;
+  grid-template-rows: min-content;
+  overflow: hidden;
+  position: relative;
+
+  @include mq($break-2-col) {
+    grid-template: auto / 41.5% 1fr;
+  }
+
+  &,
+  &__tile {
+    background-color: var(--white);
+  }
+
+  &__media {
+    aspect-ratio: 2 / 1;
+    height: 100%;
+    object-fit: cover;
+  }
+
+  &__tile {
+    align-items: flex-start;
+    display: flex;
+    flex-direction: column;
+    padding: 2rem 1.5rem;
+
+    @include mq(0 419px) {
+      align-items: center;
+      text-align: center;
+    }
+
+    @include mq($break-2-col) {
+      padding-bottom: 2rem;
+    }
+  }
+
+  &__tile-logo {
+    @include elevation(2);
+    border-radius: 0.625rem;
+    height: 3.75rem;
+    margin: 0 auto;
+    width: 3.75rem;
+  }
+
+  &__tile-eyebrow {
+    color: var(--grey-700);
+  }
+
+  &__tile-title {
+    margin: 1rem 0 2rem;
+  }
+
+  &__tile-cta {
+    margin-top: auto;
+  }
+
+  &--featured {
+    @include mq(560px) {
+      grid-column: 1 / 3;
+    }
+
+    @include mq($break-feature-col) {
+      grid-template-columns: 1fr minmax(28ch, 36.2%);
+    }
+
+    @include mq($break-2-col) {
+      grid-template-columns: 1fr 32.5% 8%;
+    }
+  }
+
+  &--featured & {
+    $spacing-s: 32px;
```

```suggestion
    $spacing-s: 2rem;
```

Prefer `rem` directly instead of `px`